gobre/recipe/template/__init__.py

```diff
@@ -18,7 +18,7 @@
 log = logging.getLogger(__name__)
```

Comment:
We still have a `__version__` and an `__author__` information in this file, which are not reflecting the reality. I would propose to remove them. What do you think?

Reply:
Removed both of them.